### PR TITLE
regenerate the sdk since the swagger is updated

### PR DESCRIPTION
**Pull request by jeffrey-ACE-zz**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please add a link to your swagger spec PR.
Also please pull latest from upstream and use generate.cmd to generate your SDK.
Finally build after generation using
msbuild build.proj /t;build /p:Scope=SDKs\ManagementPartner

this will ensure all the meta data is generated required for the SDK to be merged.

**Comment by jeffrey-ACE** [comment]
@shahabhijeet 
Swagger spec PR: https://github.com/Azure/azure-rest-api-specs/pull/2665 
Pulled from upstream.
For the other two steps, run generate.cmd and build, I did these too, no new change

**Review by shahabhijeet — APPROVED** [review]

**Comment by shahabhijeet** [comment]
@jeffrey-ACE in order to publish, you will have to bump up the version as well as add ReleaseNotes.

**Comment by jeffrey-ACE** [comment]
@shahabhijeet thank you, made the fix.

**Comment by shahabhijeet** [comment]
@jeffrey-ACE you have to generate SDK using generate.cmd or generate.ps1
We cannot accept this PR without it.

**Comment by dsgouda** [comment]
@jeffrey-ACE  Please create a `generate.ps1` script next to your `generate.cmd` and run it, this should generate the same code but with a diff in the .txt file. Please commit these changes

**Comment by dsgouda** [comment]
@jeffrey-ACE  that was expected, we are moving towards replacing the .cmd file with a .ps1 script without affecting the code generation.
Your changes look good.

**Comment by jeffrey-ACE** [comment]
@dsgouda cool, thanks, can you please also approve and merge this PR

File: src/SDKs/ManagementPartner/Management.ManagementPartner/generate.ps1 [hunk 1]
```diff
@@ -0,0 +1 @@
+powershell.exe -ExecutionPolicy Bypass -NoLogo -NonInteractive -NoProfile -File "$(split-path $SCRIPT:MyInvocation.MyCommand.Path -parent)\..\..\..\..\tools\generateTool.ps1" -ResourceProvider "managementpartner/resource-manager" -SdkDirectory "$(split-path $SCRIPT:MyInvocation.MyCommand.Path -parent)" -PowershellInvoker  -AutoRestVersion "latest"
```
Comment (dsgouda) [inline src/SDKs/ManagementPartner/Management.ManagementPartner/generate.ps1]: @jeffrey-ACE Please remove the `-SdkDirectory` argument here and generate again.
Apologize for the confusion, will fix the example soon.

**Review by dsgouda — COMMENTED** [review]
Thanks @jeffrey-ACE LGTM subject to CI builds, will merge on passing.

**Comment by jeffrey-ACE** [comment]
@dsgouda , all checks have passed, can you please merge

**Pull request merged**